packages/react-components/src/trace-explorer/trace-explorer-time-range-data-widget.tsx

```diff
@@ -32,7 +32,7 @@ export class ReactTimeRangeDataWidget extends React.Component<
     ReactTimeRangeDataWidgetProps,
     ReactTimeRangeDataWidgetState
 > {
-    private experimentDataMap: Map<string, ExperimentTimeRangeData>;
+    public experimentDataMap: Map<string, ExperimentTimeRangeData>;
```

Comment:
Should not be public.

Reply:
Can we access this with a getter method?  Or should we remove all direct ref's to this react component from outside?

Reply:
This has been changed to a getter method.